scripts/run_stochastic_process.R

```diff
@@ -25,249 +25,404 @@ do_stochastics_2021 <- function(con, test_run) {
   out_path <- "Z:/stochastic_2021_output/aggregated/"
   pre_aggregation_path <- "Z:/stochastic_2021_output/pre-aggregate/"
   log_file <- "Z:/stochastic_2021_output/log.txt"
+  output_files <- "Z:/stochastic_2021_output/output_files.csv"
```

Comment:
A bit late on this one - but I wonder if `Z:/` should be factored out, and fed into this function as `dropbox_path`

Reply:
Thanks, I've moved the full hardcoded paths out of the function body and into the main body of the script. So this isn't the dropbox root. We couldn't get drop box to sync automatically nicely on the machine so `Z:` is just a network drive, it won't try and sync to dropbox.